validation/chOpenApiValidator.go

```diff
@@ -98,7 +99,7 @@ func (chv CHValidatorImpl) ValidateRequestAgainstOpenApiSpec(httpReq *http.Reque
 	}
 
 	// Switch off the addition of schema error details to the returned error. This stops the OpenApi schema being added to errors.
-	openapi3.SchemaErrorDetailsDisabled = true
```

Comment:
Is this for debugging locally, should this be changed back?

Reply:
Good spot! Will do